staging/jenkins/Chart.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+name: jenkins
+home: https://jenkins.io/
+version: 1.7.3
+appVersion: lts
+description: Open source continuous integration server. It supports multiple SCM tools
+  including CVS, Subversion and Git. It can execute Apache Ant and Apache Maven-based
+  projects as well as arbitrary scripts.
+sources:
+- https://github.com/jenkinsci/jenkins
+- https://github.com/jenkinsci/docker-jnlp-slave
+- https://github.com/maorfr/kube-tasks
+- https://github.com/jenkinsci/configuration-as-code-plugin
```

Comment:
do we want to put in some konvoy specific sources too here since we are adding things on top of the upstream chart?

Reply:
i'm not sure if dependent charts are expected (as convention) to include sources of their parent charts.

Reply:
That list comes from upstream: https://github.com/helm/charts/blob/master/stable/jenkins/Chart.yaml